AlertaDengue/AlertaDengue/settings.py

```diff
@@ -11,10 +11,18 @@
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 import os
-from decouple import config, Csv
+import sys
+from dotenv import load_dotenv
+from pathlib import Path
 from dj_database_url import parse as db_url
 
 
+project_path = os.path.expanduser(
+    '/home/esloch/work_lab/fiocruz/AlertaDengue/'
```

Comment:
also I think you should replace this fixed string path

Reply:
Updated according to the documentation; python-dotenv 0.10.3:
env_path = Path('.') / '.env'
load_dotenv(dotenv_path=env_path)

Thanks @xmnlab.